<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>511</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>511</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PPC 5 Character Editor - Untitled</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="grp_editing">
        <property name="title">
         <string>Editing</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QTreeView" name="e_Editor">
           <attribute name="headerCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="headerDefaultSectionSize">
            <number>300</number>
           </attribute>
           <attribute name="headerHighlightSections">
            <bool>false</bool>
           </attribute>
           <attribute name="headerMinimumSectionSize">
            <number>0</number>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QScrollBar" name="e_timelineScroll">
           <property name="maximum">
            <number>600</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="grp_properties">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>110</height>
            </size>
           </property>
           <property name="title">
            <string/>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QGroupBox" name="grp_keyframe">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>9999</height>
               </size>
              </property>
              <property name="title">
               <string>Keyframe Properties</string>
              </property>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="l_easingin">
                 <property name="minimumSize">
                  <size>
                   <width>64</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Ease in:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="e_easingin">
                 <item>
                  <property name="text">
                   <string>Linear</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Cubic</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Quadratic</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Sinusoidal</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Exponential</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>None</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="l_value">
                 <property name="minimumSize">
                  <size>
                   <width>64</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Value:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="e_value">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="l_easingout">
                 <property name="minimumSize">
                  <size>
                   <width>64</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Ease out:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="e_easingout">
                 <item>
                  <property name="text">
                   <string>Linear</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Cubic</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Quadratic</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Sinusoidal</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Exponential</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>None</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="grp_charinfo">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>999</height>
               </size>
              </property>
              <property name="title">
               <string>Character Properties</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="spacing">
                <number>6</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <layout class="QFormLayout" name="layout_col1">
                 <property name="sizeConstraint">
                  <enum>QLayout::SetDefaultConstraint</enum>
                 </property>
                 <property name="fieldGrowthPolicy">
                  <enum>QFormLayout::ExpandingFieldsGrow</enum>
                 </property>
                 <property name="labelAlignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="leftMargin">
                  <number>6</number>
                 </property>
                 <property name="topMargin">
                  <number>9</number>
                 </property>
                 <property name="rightMargin">
                  <number>6</number>
                 </property>
                 <property name="bottomMargin">
                  <number>6</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QLabel" name="l_name">
                   <property name="minimumSize">
                    <size>
                     <width>88</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Character Name:</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="e_name">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>256</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>256</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Monopoly Man</string>
                   </property>
                   <property name="maxLength">
                    <number>255</number>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="l_dropset">
                   <property name="minimumSize">
                    <size>
                     <width>88</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Dropset:</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QGraphicsView" name="e_dropset">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>256</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>256</width>
                     <height>20</height>
                    </size>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QCheckBox" name="e_asymmetrical">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QLabel" name="l_asymmetrical">
                   <property name="text">
                    <string>Use asymmetrical images in the character select</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QFormLayout" name="layout_col2">
                 <property name="labelAlignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="leftMargin">
                  <number>6</number>
                 </property>
                 <property name="topMargin">
                  <number>9</number>
                 </property>
                 <property name="rightMargin">
                  <number>6</number>
                 </property>
                 <property name="bottomMargin">
                  <number>6</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QLabel" name="l_trait">
                   <property name="text">
                    <string>AI Trait:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QComboBox" name="e_trait">
                   <item>
                    <property name="text">
                     <string>Normal</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="l_aidifficulty">
                   <property name="text">
                    <string>AI Difficulty:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QPushButton" name="b_managetextures">
                   <property name="text">
                    <string>Manage Textures...</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QComboBox" name="e_aidifficulty">
                   <property name="currentIndex">
                    <number>1</number>
                   </property>
                   <item>
                    <property name="text">
                     <string>Easy</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Normal</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Hard</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Core</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignTop">
       <widget class="QGroupBox" name="grp_preview">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>402</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>9999</height>
         </size>
        </property>
        <property name="title">
         <string>Preview</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QGraphicsView" name="graphicsView">
           <property name="minimumSize">
            <size>
             <width>192</width>
             <height>384</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>192</width>
             <height>384</height>
            </size>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="grp_control">
           <property name="title">
            <string>Playback Controls</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="b_play">
              <property name="text">
               <string>Play</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="b_stop">
              <property name="text">
               <string>Stop</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_Recent_Files">
     <property name="title">
      <string>&amp;Recent Files</string>
     </property>
     <addaction name="action1_Untitled"/>
     <addaction name="action2_Untitled"/>
     <addaction name="action3_Untitled"/>
     <addaction name="action4_Untitled"/>
     <addaction name="action5_Untitled"/>
     <addaction name="action6_Untitled"/>
     <addaction name="action7_Untitled"/>
     <addaction name="action8_Untitled"/>
     <addaction name="action9_Untitled"/>
     <addaction name="action10_Untitled"/>
    </widget>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="menu_Recent_Files"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="action1_Untitled">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>1. Untitled</string>
   </property>
   <property name="shortcut">
    <string>Alt+1</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action2_Untitled">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>2. Untitled</string>
   </property>
   <property name="shortcut">
    <string>Alt+2</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action3_Untitled">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>3. Untitled</string>
   </property>
   <property name="shortcut">
    <string>Alt+3</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action4_Untitled">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>4. Untitled</string>
   </property>
   <property name="shortcut">
    <string>Alt+4</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action5_Untitled">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>5. Untitled</string>
   </property>
   <property name="shortcut">
    <string>Alt+5</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action6_Untitled">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>6. Untitled</string>
   </property>
   <property name="shortcut">
    <string>Alt+6</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action7_Untitled">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>7. Untitled</string>
   </property>
   <property name="shortcut">
    <string>Alt+7</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action8_Untitled">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>8. Untitled</string>
   </property>
   <property name="shortcut">
    <string>Alt+8</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action9_Untitled">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>9. Untitled</string>
   </property>
   <property name="shortcut">
    <string>Alt+9</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action10_Untitled">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>10. Untitled</string>
   </property>
   <property name="shortcut">
    <string>Alt+0</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
